libraries/Wire/src/Wire.cpp

```diff
@@ -455,6 +455,14 @@ uint8_t TwoWire::requestFrom(uint16_t address, uint8_t quantity, uint8_t sendSto
     return requestFrom(address, static_cast<size_t>(quantity), static_cast<bool>(sendStop));
```

Comment:
everywhere that you see `static_cast<size_t>(quantity)` needs to change to `static_cast<uint8_t>(quantity)`